gadget-container/entrypoint.sh

```diff
@@ -17,12 +17,11 @@
 set -e
 
 if [ ! -r /host/etc/os-release ] ; then
-  echo "$0 must be executed in a pod with access to the host via /host" >&2
-  exit 1
+  echo "os-release information not available. Some features could not work"
```

Comment:
Could you add a comment saying that os-release is not available on all distros such as NixOS?